setup.py

```diff
@@ -8,6 +8,29 @@
 with open("LICENSE") as f:
     _license = f.read()
 
+dev_packages = [
+    "black",
+    "isort",
+    "pytest",
+]
+
+required_packages = [
+    "click",
+    "descartes",
+    "geopandas",
+    "ipykernel",
+    "matplotlib",
+    "numpy",
+    "pandas",
+    "pyarrow",
+    "python-dotenv",
+    "requests",
+    "rtree",
+    "shapely",
+    "stringcase",
+    "tqdm",
+]
+
 setup(
     name=name,
     version="0.0.0",
```

Comment:
shouldn't we be maintaining/updating versions?

Reply:
I've generally been pretty relaxed here - the trade-off is that for general public use, we don't want to overly constrain package versions, otherwise we risk adding complexity/incompatibility for practical use - so if there's a problem we know about, we should pin it. If we think the risk is pretty low, we should let it float. This differs from internal production apps, which need much stricter pinning via a separate process.

Reply:
Sounds good